### ISSUE TITLE
Can not install via pip or from latest git version

### ISSUE THREAD

**Issue by theChaosCoder** [Irrational-Encoding-Wizardry/vs-dehalo#4]
```
pip install vs-dehalo
ERROR: Could not find a version that satisfies the requirement vs-dehalo (from versions: none)
ERROR: No matching distribution found for vs-dehalo

```
```
pip install git+https://github.com/Irrational-Encoding-Wizardry/vs-dehalo.git
ERROR: Could not find a version that satisfies the requirement vs-denoise>=1.0.0 (from vsdehalo) (from versions: none)
ERROR: No matching distribution found for vs-denoise>=1.0.0
```

**Comment by LightArrowsEXE** [comment]
Yeah, this package depends on vs-denoise right now which isn't public yet. Not sure why @Setsugennoao added install instructions already. This issue should resolve itself once vs-denoise becomes publicly available, though!

**Comment by Setsugennoao** [comment]
Sorry 'bout that

**Comment by Setsugennoao** [comment]
Fixed in v0.1.1
